packages/openneuro-components/src/count-toggle/CountToggle.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import './count-toggle.scss'
+import { Tooltip } from '../tooltip/Tooltip'
+import { Button } from '../button/Button'
+import { Modal } from '../modal/Modal'
+import { UserModalInner } from '../modal/UserModalInner'
+
+export interface CountToggleProps {
+  label?: string
+  icon?: string
+  disabled?: boolean
+  tooltip?: string
+  onClick?: () => void
+  lock?: boolean
+  displayOptions: boolean
+  setDisplayOptions: (boolean) => void
+  count: number
+}
+export const CountToggle = ({
+  label,
+  icon,
+  disabled,
+  tooltip,
+  onClick,
+  displayOptions,
+  setDisplayOptions,
+  count,
+}: CountToggleProps) => {
+  const toggleClick = () => {
+    onClick()
+    setDisplayOptions(!displayOptions)
+  }
```

Comment:
This can belong to the parent component, so that only toggleClick and displayOptions need to be passed in.

I think for this application, it would also be good to rename displayOptions to `clicked` or `showClicked`.